source/elements/oneDAL/source/introduction.rst

```diff
@@ -1,3 +1,20 @@
 ============
 Introduction
 ============
+
+oneDAL is library that provides building blocks covering all stages of data analytics: data acquisition
+ from a data source, preprocessing, transformation, data mining, modeling, validation, and decision making.
+
+Library consists of the following major components: Data Management, Algorithms, and Services
+
+The Data Management component includes classes and utilities for data acquisition, initial preprocessing
+ and normalization, for data conversion into numeric formats done by one of supported Data Sources,
+ and for model representation.
```

Comment:
When we have the final names of the respective types we will have to add them as examples of the content

Reply:
yes, sure. will write down this as comment for future updates